lib/rubycritic/rake_task.rb

```diff
@@ -71,7 +71,7 @@ def run_task
 
     def print_starting_up_output
       puts "\n\n!!! Running `#{name}` rake command\n"
-      puts "!!! Inspecting #{paths} #{options.empty? ? '' : "with options #{options}"}\n\n"
+      puts "!!! Inspecting #{paths} #{"with options #{options}" unless options.empty?}\n\n"
```

Comment:
@faisal maybe changing this to single quotes helps the reading? what do you think?

```suggestion
      puts "!!! Inspecting #{paths} #{'with options #{options}' unless options.empty?}\n\n"
```